bertinNDAYISHIMIYE/capstoneProject-MyBrand
# ft-tests-setUp

###What does this PR do?

>add necessary packages
>add db config modification
###Description of the tasks to be completed
>Set up tests environment
###How should this be tested manually?

>Clone this repo with: https://github.com/bertinNDAYISHIMIYE/capstoneProject-MyBrand
>Checkout to ft-tests-setUp branch
>Make sure node.js is installed
>Run command npm init for npm initialisation
>Run npm i to install different packages
>Use postman as Third party software to test the end point

###Any background context you want to provide?

N/P
###Relavant trello card
https://trello.com/c/bZBHJmUl
